DSMAJCEexamples/src/main/java/DSMAJCE.java

```diff
@@ -0,0 +1,119 @@
+import com.fortanix.sdkms.jce.provider.SdkmsJCE;
```

Comment:
Conceptually, I want to have this code be "Programmatic Documentation" on how to use the library.  It should have lots of comments and describe the steps to setup the DSM for it's operation.  It should show a wide variety of the use cases supported, multiple ways to configure and have functions that can be used by customers to create test benches of their business use cases.